Cannot open include file DriverEnums.h

I tried to create a new filament Engine:
`filament::Engine* m_Engine = filament::Engine::create( );`
but I got this error:
`2>e:\brawler\external_projects\thirdparty\filament\include\filament\fence.h(24): fatal error C1083: Cannot open include file: 'filament/driver/DriverEnums.h': No such file or directory`
The folder indeed does not have DriverEnums.h, neither can I find it in the latest github branch.

Now I am getting an error in driverenums.h:
`2>e:\brawler\external_projects\thirdparty\filament\include\filament\driver\driverenums.h(110): error C2059: syntax error: 'constant'`
It's the ERROR enum in this class:
```
enum class FenceStatus : int8_t {
    ERROR = -1,                 //!< An error occured. The Fence condition is not satisfied.
    CONDITION_SATISFIED = 0,    //!< The Fence condition is satisfied.
    TIMEOUT_EXPIRED = 1,        //!< wait()'s timeout expired. The Fence condition is not satisfied.
};
```

--- a/filament/include/filament/Frustum.h
+++ b/filament/include/filament/Frustum.h
@@ -25,6 +25,8 @@
 #include <math/vec4.h>
 #include <math/vec2.h>
 
+#include <utils/unwindows.h> // Because we define NEAR and FAR in the Plane enum.
+
 namespace filament {
 
 namespace details {

--- a/libs/filabridge/include/filament/driver/DriverEnums.h
+++ b/libs/filabridge/include/filament/driver/DriverEnums.h
@@ -24,6 +24,8 @@
 #include <stddef.h>
 #include <stdint.h>
 
+#include <utils/unwindows.h> // Because we define ERROR in the FenceStatus enum.
+
 namespace filament {
 
 /**

--- a/libs/utils/CMakeLists.txt
+++ b/libs/utils/CMakeLists.txt
@@ -17,6 +17,7 @@ set(DIST_HDRS
         ${PUBLIC_HDR_DIR}/${TARGET}/EntityInstance.h
         ${PUBLIC_HDR_DIR}/${TARGET}/EntityManager.h
         ${PUBLIC_HDR_DIR}/${TARGET}/CString.h
+        ${PUBLIC_HDR_DIR}/${TARGET}/unwindows.h
 )
 
 set(SRCS

